src/additional/ishexcolor.js

```diff
@@ -0,0 +1,3 @@
+$.validator.addMethod( "ishexcolor", function( value, element ) {
+	return this.optional( element ) || /^#(?:[0-9a-f]{6}|[0-9a-f]{3})$/i.test( value );
+}, "This is not hex format color" );
```

Comment:
Please change this message to something like: `Please specify a valid hex color value`